packages/bp/src/migrations/v12_28_0-1662113636-bump_messaging.ts

```diff
@@ -0,0 +1,21 @@
+import * as sdk from 'botpress/sdk'
+import { Migration, MigrationOpts } from 'core/migration'
+import { runMessagingMigration } from 'orchestrator/messaging-server'
+
+const migration: Migration = {
+  info: {
+    description: 'Migrates the messaging database from 1.2.0 to 1.2.3',
+    target: 'core',
+    type: 'database'
+  },
+
+  up: async ({ metadata }: MigrationOpts): Promise<sdk.MigrationResult> => {
+    return runMessagingMigration('up', '1.2.3', metadata.isDryRun)
+  },
+
+  down: async ({ metadata }: MigrationOpts): Promise<sdk.MigrationResult> => {
+    return runMessagingMigration('down', '1.2.0', metadata.isDryRun)
+  }
```

Comment:
Without this migration, it's impossible to let's say, migrate from 12.27 to 12.28 and then back to 12.27.